api/common_windows.go

```diff
@@ -1,4 +1,8 @@
 package api
 
 // MinVersion represents Minimum REST API version supported
-const MinVersion string = "1.25"
+// Technically the first daemon API version released on Windows is v1.25 in
+// engine version 1.13. However, some clients are explicitly using downlevel
+// APIs (eg docker compose v2.1) and that is just too restrictive. Hence
```

Comment:
We may want to explain this is the `docker-compose` 2.1 _file_ format, not the version of docker compose